### ISSUE TITLE
Totalstation Pointing to Robot / Tracking

### ISSUE THREAD

**Issue by robin271828** [leggedrobotics/grand_tour_box#92]
- [x] Working pointing and recovering the track
- [ ] Time synchronization coordinate what to do with the constant offset with Rik 
- [x] Running open3dslam on the robot
- [ ] Alignment of Totalstation frame and robot frame (Blocked by Julian)
- [ ] Make a nice totalstation script nicer
- [ ] Write a readme about using the totalstation
- [ ] Integration of totalstation in RViZ with nice visualization (nice to have)
